setup.py

```diff
@@ -11,25 +11,32 @@
         ]
       },
       install_requires=[],
-      url="http://en.wikipedia.org/wiki/Cpplint",
-      download_url="https://github.com/cpplint/cpplint",
-      keywords=["lint", "python", "c++"],
+      url='http://en.wikipedia.org/wiki/Cpplint',
+      download_url='https://github.com/cpplint/cpplint',
+      keywords=['lint', 'python', 'c++'],
       maintainer = 'cpplint Developers',
       maintainer_email = 'see_github@nospam.com',
-      classifiers=["Programming Language :: Python",
-                   "Programming Language :: Python :: 2",
-                   "Programming Language :: Python :: 2.7",
-                   "Programming Language :: Python :: 3",
-                   "Programming Language :: Python :: 3.2",
-                   "Programming Language :: Python :: 3.3",
-                   "Programming Language :: Python :: 3.4",
-                   "Programming Language :: Python :: 3.5",
-                   "Programming Language :: Python :: 3.6",
-                   "Programming Language :: Python :: 3.7",
-                   "Programming Language :: C++",
-                   "Development Status :: 5 - Production/Stable",
-                   "Environment :: Console",
-                   "Topic :: Software Development :: Quality Assurance",
-                   "License :: Freely Distributable"],
-      description="An automated checker to make sure a C++ file follows Google's C++ style guide",
-      long_description=open('README.rst').read())
+      classifiers=['Programming Language :: Python',
+                   'Programming Language :: Python :: 2',
+                   'Programming Language :: Python :: 2.7',
+                   'Programming Language :: Python :: 3',
+                   'Programming Language :: Python :: 3.2',
+                   'Programming Language :: Python :: 3.3',
```

Comment:
Should we drop 3.2 and 3.3 as they are EOL?

Reply:
Have edited the commits to remove those strings as well as drop from the matrix. Thanks.